CrisisTextLine/Elmer
# Notify First Shift working; sending emails properly.

#### What's this PR do?
Fixes NotifyFirstShift so that emails are sent to users who are new graduates and have signed up for a first shift.
#### Where should the reviewer start?
jobs/scheduling/NotifyFirstShift and test/notifyFirstShift.
#### How should this be manually tested?
"npm test" or use When I Work test environment to sign up for a first shift and see the e-mail generated.
#### Any background context you want to provide?
#### What are the relevant tickets?
Kind of a prelude to getting [INT-96](https://admin.crisistextline.org/jira/browse/INT-96), [INT-101](https://admin.crisistextline.org/jira/browse/INT-101), and [INT-102](https://admin.crisistextline.org/jira/browse/INT-102) done.
#### Questions:
